### PR TITLE
Fix right to left issues

### LINKED ISSUE #72: Add option to RTL file
header :)

### LINKED ISSUE #126: Adding an option to set RTL worksheet
Header

### PR DESCRIPTION
**Pull request by alitaheri**
Adds a new property to sheet view. And fixes `readingOrder` as it wasn't doing anything for 2 reasons:

1. The values were set wrong. 
2. The keys were `l2r` and `r2l` instead of the document `ltr` and `rtl` ones.

Closes #72
Closes #126

### CODE REVIEW

File: lib/xlsx/xform/sheet/sheet-view-xform.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ utils.inherits(SheetViewXform, BaseXform, {
         xmlStream.addAttribute(name, value);
       }
     };
+    add('rightToLeft', '1', model.rtl === true);
```
Comment (guyonroche) [inline lib/xlsx/xform/sheet/sheet-view-xform.js]: there needs to be a corresponding line in the parseOpen function or it won't be able to read files properly

Comment (guyonroche) [inline lib/xlsx/xform/sheet/sheet-view-xform.js]: Also - could you add this property to the unit test:
in spec/unit/xlsx/xform/sheet/sheet-view-xform.spec.js

File: lib/xlsx/xform/style/alignment-xform.js [hunk 2]
```diff
@@ -56,10 +56,10 @@ var validation = {
     return Math.max(0, value);
   },
   readingOrderValues: [
-    {k: 'r2l', v: 1},
-    {k: 'l2r', v: 2},
-    {k: Enums.ReadingOrder.RightToLeft, v: 1},
-    {k: Enums.ReadingOrder.LeftToRight, v: 2}
+    {k: 'rtl', v: 2},
```
Comment (guyonroche) [inline lib/xlsx/xform/style/alignment-xform.js]: Would you be able to add a unit test for these changes?
in spec/unit/xlsx/xform/style/alignment-xform.spec.js

**Comment by alitaheri** [comment]
@guyonroche I made the requested changes.

I also renamed `rtl` to `rightToLeft` to make it consistent with other props.

**Comment by pouyakary** [comment]
@alitaheri what a small world!

**Pull request merged**